sphinx/source/operators/resource_usage.rst

```diff
@@ -0,0 +1,17 @@
+Resource Usage
+==============
+
+CPU
+---
+
+A CCF application currently consists of two threads. A host thread manages sockets and files, and handles communication with the enclave via ringbuffers.
+And enclave thread contains the TLS termination, all cryptography, and the application and key value code. It communicates with the host via ringbuffers too.
```

Comment:
Typo: should be "**An** enclave thread", or "The".

Reply:
Yes, thank you!